fpm/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+bundle install --path "${HOME}/bundles/${JOB_NAME}"
+
+cd "recipes/${RECIPE_NAME}/"
+
+bundle exec fpm-cook --target deb
+
+rsync -v pkg/*.deb deploy@apt-1.management:/tmp
+
```

Comment:
That makes sense. @deanwilson pointed out that we could use Jenkins archiving of artefacts which seems like a good way to go so we've updated this script to push stuff into a build directory which Jenkins will be configured to archive the contents of.